packages/custom-field/src/vaadin-custom-field.d.ts

```diff
@@ -72,6 +72,7 @@ export interface CustomFieldEventMap extends HTMLElementEventMap, CustomFieldCus
  * `helper-text`        | The slotted helper text element wrapper
  * `error-message`      | The slotted error message element wrapper
  * `required-indicator` | The `required` state indicator element
+ * `input-wrapper`      | The slotted input elements wrapper
```

Comment:
Should we reconsider the naming since none of the other "wrapper" parts have a "-wrapper" suffix (`helper-text`, `error-message`)